plugins/basic/sg_bootstrap.py

```diff
@@ -21,10 +21,10 @@ def log(msg):
 
     print(message)
 
-# We only initialize Shotgun once in the client process
+# We only initialize Shotgun once in the bootstrap process
```

Comment:
There is no such thing as a bootstrap process. Just keep `We only initialize Shotgun once`